<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConfig</class>
 <widget class="QDialog" name="DialogConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Config</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_RigModel">
     <property name="text">
      <string>Model</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox_rigModel"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_RigPath">
     <property name="text">
      <string>Path</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_RigPath</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_RigPath"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_serialSpeed">
     <property name="text">
      <string>Speed</string>
     </property>
     <property name="buddy">
      <cstring>comboBox_serialSpeed</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboBox_serialSpeed"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_RefreshRate">
     <property name="text">
      <string>Refresh Rate</string>
     </property>
     <property name="buddy">
      <cstring>spinBox_RefreshRate</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="spinBox_RefreshRate">
     <property name="minimum">
      <number>50</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="singleStep">
      <number>50</number>
     </property>
     <property name="value">
      <number>200</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_civAddr">
        <property name="text">
         <string>CI-V Address</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBox_fullPoll">
        <property name="text">
         <string>Full polling</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_civAddr">
        <property name="toolTip">
         <string>CI-V address (decimal, Icom only)</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_RigPath</tabstop>
  <tabstop>comboBox_serialSpeed</tabstop>
  <tabstop>spinBox_RefreshRate</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
